packages/wrangler/src/__tests__/versions/secrets/put.test.ts

```diff
@@ -54,41 +55,60 @@ describe("versions secret put", () => {
 		expect(std.err).toMatchInlineSnapshot(`""`);
 	});
 
-	// For some reason, this always hangs. Not sure why
```

Comment:
This was broken because you cannot call `useMockStdin()` from within an `it()` clause since itself calls things like `beforeEach()`.